WebKitBrowser/CMakeLists.txt

```diff
@@ -15,39 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.3)
```

Comment:
Yes, will add this, no problem

Reply:
I meant to keep the whole section here: 

```
find_package(WPEFramework)

list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../cmake/")

include(CmakeHelperFunctions)

# Library installation section
string(TOLOWER ${NAMESPACE} STORAGE_DIRECTORY)

if(BUILD_REFERENCE)
    add_definitions(-DBUILD_REFERENCE=${BUILD_REFERENCE})
endif()
````

otherwise, build is broken for us:

```
CMake Error at CMakeLists.txt:99 (find_package):
  By not providing "FindPlugins.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Plugins", but
  CMake did not find one.

  Could not find a package configuration file provided by "Plugins" with any
  of the following names:

    PluginsConfig.cmake
    plugins-config.cmake

  Add the installation prefix of "Plugins" to CMAKE_PREFIX_PATH or set
  "Plugins_DIR" to a directory containing one of the above files.  If
  "Plugins" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
```

unless it is something that will be fixed with changes in Thunder cmake templates?

Reply:
Ok, we looked at it and we have no problem adding that section, I'll add it back

Reply:
Thanks!